src/app/core/ui/ui.module.ts

```diff
@@ -0,0 +1,35 @@
+/*
+ * Lumeer: Modern Data Definition and Processing Platform
+ *
+ * Copyright (C) since 2017 Answer Institute, s.r.o. and/or its affiliates.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {NgModule} from '@angular/core';
+import {CommonModule} from '@angular/common';
+import {SharedModule} from '../../shared/shared.module';
+import {DocumentUiService} from './document-ui.service';
+
+@NgModule({
+  imports: [
+    CommonModule,
+    SharedModule
+  ],
+  providers: [
+    DocumentUiService
+  ]
+})
+export class UiModule {
```

Comment:
You don't really need this module. Just define `DocumentUiService` like this:
```typescript
@Injectable({
  providedIn: 'root',
})
```

Reply:
I thought that you preferred to have fine grained modules... I can easily delete it, I do not need it to be in a module.

Reply:
Sure, we want to split application into small modules. But there is no point in creating a module for a single service. Moreover, since Angular 6 it does not need to be declared in any module.

Reply:
My idea was that we might end up having LinkUiService etc. as well. Hence the module. Is the module some obstacle or what?